src/js/js-loader.ts

```diff
@@ -69,4 +70,14 @@ function isExcludedResource(resource: string): boolean {
     return /angular-translate[\/\\]dist[\/\\]angular-translate\.js$/.test(resource);
 }
 
-export = jsLoader;
\ No newline at end of file
+function parseOptions(version: number, query: any) {
```

Comment:
Why are you not using [loader-utils](https://github.com/webpack/loader-utils) to parse the query. As I remember, the query can also be a string, in which case this will fails?